src/data/corpus.cpp

```diff
@@ -266,6 +271,14 @@ CorpusBase::batch_ptr Corpus::toBatch(const std::vector<Sample>& batchVector) {
     }
```

Comment:
Would you mind updating the comment with the `maxDims` variable? It is not obvious from the code what it means, and why it is OK to round it to multiples of 8. (Based on the variable name, it seems you are changing the hidden dimension, which would not work.) And if the variable is misleadingly named, then would you mind naming it better (according to your change below, it seems related to batch width = max # words)? Thanks!

Reply:
It seemed to be the max # of words in a batch. I just kept the same name but I can rename to `maxWordsInBatch`

Reply:
(renamed in recent commits)